### ISSUE TITLE
Total reward = 10 for Connect4 

### ISSUE THREAD

**Issue by akopf82** [werner-duvaud/muzero-general#93]
The total reward of Connct4 iterates to 10 and keeps constant afterwards. Even after 100k training steps the game performance seems ramdomized. 
[
<img width="469" alt="tmp" src="https://user-images.githubusercontent.com/39181776/101328746-b808a900-3870-11eb-85cf-2c2ceb6d7238.png">
](url)

**Comment by werner-duvaud** [comment]
Hi,
I understand this is confusing, for two player games, the `total reward` is the sum of the rewards over the entire game. at tic tac toe there is necessarily a winner receiving a reward of 10. This explains what we see.
On the other hand, the `Muzero reward` and `opponent reward` plots are interesting, I have the impression that muzero wins a little more frequently towards the end of your run.

**Comment by akopf82** [comment]
Ok, the reward calculation/definition is clear to me. However, i've performed several tests (different games) using the initial configuration, and i always get mean values (Total reward/mean values [plot 2 / tictactoe]) alternating around 0.
<img width="783" alt="tmp" src="https://user-images.githubusercontent.com/39181776/104810935-538a2380-57f8-11eb-8d58-46faba7a8af7.png">
The loss calculation looks realistic to me (also compared to your screenshots), but the reward doesn't show progress over 10k or even 100k iterations. 
This observation also applies to the following modifications:
- mutliple GPU / one GPU
- varying amout of num_workers
- reduzed self.support_size
- "resnet" / "fullyconnected"
- "Adam" or "SGD"
- self.root_exploration_fraction 0.25 / 0.3
System: Python 3.7.6 , CentOS, Xeon(R) Gold + 2 x V100
